README.md

```diff
@@ -22,30 +24,84 @@ To use it in Maven, add the following to your POM:
 ```
 [//]: # ({x-version-update-end})
 
-Dependency Dashboard
-------------
+When you use the Libraries BOM, you don't specify individual library
+versions and your application runs on a consistent set of the dependencies.
+
+## Libraries in Scope
+
+The content of the Libraries BOM consists of 2 categories: Google Cloud Java
+client libraries and core Google dependency libraries.
```

Comment:
I think this doesn't answer the question why some client libraries are not in the BOM, such as some of the ones that are not GA.

Reply:
Added a section for preview state libraries.